lib/setupmanagers/hckclient.rb

```diff
@@ -72,11 +72,12 @@ def set_machine_ready
       raise ClientRunError, "Couldn't set #{@name} state to Ready"
     end
 
-    def run_pre_test_commands
+    def run_pre_tests_commands
```

Comment:
- If you rename a method you should do so for the JSON too.
- The differences of "test" and "tests" are too subtle. It is not really related to tests so better avoid using the word "tests". For example, you can name it "post_start".